### DISCUSSION TITLE
How do people identify USWDS components in their wireframes

### DISCUSSION THREAD

**Discussion by mgifford** [uswds/uswds#5333] [Accessibility]
I brought this up in Slack, but would love a broader discussion of this here. 

Are people identifying what elements are part of the USWDS when creating wireframes. I'm a big fan of [Accessibility Annotations](https://medium.com/designing-atlassian/create-accessible-designs-using-the-figma-a11y-annotation-kit-35371f00dac5), but in using a design system, much of what needs to be identified is just what components are used. 

We've seen many instances where front-end developers choose multiple ways to implement something that looks like a USWDS component, while altering the semantics. It's possible that they just didn't realize that there was a component they were supposed to mirror, so just used whatever tools were fastest to craft up.

**Comment by mejiaj** [comment]
@mgifford can you share some examples of your annotations? What part of the process from design → development phase does this happen?

**Comment by mgifford** [comment]
I don't have a visual for it. We haven't implemnted this. Could just start from the triangles from the USWDS logo.

Ultimately though, it is about changing the process so that USWDS components are explicitly named in designs, so that they aren't forgotten or overlooked in the implementation. 

I do think it is common for designers to overlook the design components and build from scratch much of th etime. I also think it's common for front-end folks to not know that a USWDS design component was even considered.

Making explicit makes the transitions easier.
